.github/workflows/ci-ecr.yaml

```diff
@@ -86,6 +86,16 @@ jobs:
           LATEST_GIT_SHA=$(git ls-remote origin HEAD | cut -f 1)
           IMAGE_TAG_ARGS="-t ${ECR_REGISTRY}/${ECR_REPOSITORY}:${IMAGE_TAG}"
 
+          check_image_in_ecr() {
+            aws ecr describe-images --repository-name="${1}" --image-ids=imageTag="${2}"
+            return $?
```

Comment:
No need to explicitly return the exit code - the value of a shell function is its exit status already.
```suggestion
```

Reply:
Will do, thanks